frontend/src/App.css

```diff
@@ -93,9 +93,42 @@ body {
   background-color: #0a2ea4;
 }
 
+.check-box {
+  transform: scale(1.5);
+  margin-right: 5%;
+}
+
 .todo-form {
   display: flex;
   width: 200%;
   vertical-align: top;
   text-align: center;
 }
+
+.category-items {
+  float: left;
```

Comment:
We no longer use float to define a layout in CSS. If you need to arrange some items in a row or a column, use flexbox instead. You can use [this](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) guide to learn about flex box.